### PR TITLE
Feature/slider

### PR DESCRIPTION
**Pull request by tomsvogel**

### CODE REVIEW

File: components/SpeakerSlide.js [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            <SlideFilter active={active}>
+                <SlideInfo active={active} style={{opacity: active ? 1 : 0}}>
+                    {speaker.isModerator && <Moderator>Presenter</Moderator>}
+                    <SlideHeadline>
+                        {speaker.name}
+                    </SlideHeadline>
+                    <SlideSubHeadline>
+                        {speaker.title}
+                    </SlideSubHeadline>
+                    <SlideTeaser dangerouslySetInnerHTML={{__html: speaker.teaser}} />
```
Comment (danrot) [inline components/SpeakerSlide.js]: Just curious, why is this necessary?

File: pages/slider.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import fetch from 'isomorphic-fetch';
+
+import Page from '../components/Page';
+import SpeakerSlider from '../components/SpeakerSlider';
+import config from '../config';
+
+export default class Index extends React.PureComponent {
```
Comment (danrot) [inline pages/slider.js]: `Slider`, and what is this slide for anyway? Is there a separate page only with the speakers?

File: static/js/tracking.header.js [hunk 3]
```diff
@@ -2,26 +2,26 @@
...
-    h._hjSettings = {hjid: 325775, hjsv: 5};
-    a = o.getElementsByTagName('head')[0];
-    r = o.createElement('script');
-    r.async = 1;
-    r.src = t + h._hjSettings.hjid + j + h._hjSettings.hjsv;
-    a.appendChild(r);
-})(window, document, '//static.hotjar.com/c/hotjar-', '.js?sv=');
-/* eslint-enable */
\ No newline at end of file
+// (function(h, o, t, j, a, r) {
```
Comment (danrot) [inline static/js/tracking.header.js:16]: Don't we need that anymore?

**Comment by danrot** [comment]
Fixed a few things, but I also don't see any arrows like in the design, any idea why is that?

File: static/js/tracking.header.js (hunk 3 above)
Comment (tomsvogel) [reply static/js/tracking.header.js:16]: guntram told me there is a new code but i did not get one, and if we do not handle on this results we only slow down page load.
